tools/deployment/package-helm/templates/database-data-pv.yaml

```diff
@@ -0,0 +1,8 @@
+{{- include "clp.createLocalPv" (dict
+  "root" .
```

Comment:
I asked AI about this and got a reasonable answer, but do you know where I can find the docs for what `.` does in this context?

Reply:
this might be the best: https://pkg.go.dev/text/template#:~:text=Execution%20of%20the%20template%20walks%20the%20structure%20and%20sets%20the%20cursor%2C%20represented%20by%20a%20period%20%27.%27%20and%20called%20%22dot%22%2C%20to%20the%20value%20at%20the%20current%20location%20in%20the%20structure%20as%20execution%20proceeds.

the `.` represents the context of the template expansion